packages/react-formio/src/components/pagination/pagination.component.spec.tsx

```diff
@@ -44,16 +44,48 @@ describe("Pagination", () => {
 
   it("should call gotoPage() callback", () => {
     const gotoPageSpy = jest.fn();
-    const { queryAllByTestId } = render(
+    let page: number;
+  
+    const { container, queryAllByTestId } = render(
       <Sandbox gotoPage={gotoPageSpy} {...Sandbox.args} />
     );
+    const wrappersBtn = container.getElementsByClassName("page-item");
     const paginationBtn = queryAllByTestId("pagination-button");
-    const btnPage = paginationBtn.filter(
+
+    const buttonsPage = paginationBtn.filter(
       (btn) => btn.textContent !== "Previous" && btn.textContent !== "Next"
     );
-    btnPage.forEach((btn) => {
-      fireEvent.click(btn);
-      expect(gotoPageSpy).toHaveBeenCalled();
+
+    expect(
+      Array.prototype.every.call(wrappersBtn, (wrap) =>
+        wrap.classList.contains("active")
+      )
+    ).toBeFalsy();
+    expect(
+      Array.prototype.every.call(wrappersBtn, (wrap) =>
+        wrap.classList.contains("disabled")
+      )
+    ).toBeFalsy();
+    expect(
+      Array.prototype.some.call(wrappersBtn, (wrap) =>
+        wrap.classList.contains("active")
+      )
+    ).toBeTruthy();
+    expect(
+      Array.prototype.some.call(wrappersBtn, (wrap) =>
```

Comment:
I'm not sure for this test, maybe in some case buttons "previous" and "next" have not class name "disabled"